node/modules/storageminer.go

```diff
@@ -463,39 +465,56 @@ func NewLogsDB(logsSqlDB *LogSqlDB) *db.LogsDB {
 	return db.NewLogsDB(logsSqlDB.db)
 }
 
+// We start Boost in this way because we want to ensure Boost is started after the legacy markets Provider.
+// We want to do this so that the index provider callback registered by Boost can overwrite the index
+// provider callback registered by legacy markets as legacy markets does NOT know about Boost storage deals.
```

Comment:
Is this comment still correct? It looks like we're starting the index provider in `HandleIndexProvider`

Reply:
Removed.

Reply:
And added a comment to `HandleIndexProvider`.